apps/condo/domains/ticket/tasks/closeCompletedTickets.js

```diff
@@ -27,7 +27,7 @@ const closeCompletedTickets = async (defaultLimit = 100) => {
     // Therefore, you should limit updating tickets for each organization at one time if there are a lot of them.
     // With the help of the feature flag, you can manage this restriction.
     const limitByOrganization = await featureToggleManager.getFeatureValue(
-        context, MAX_COUNT_COMPLETED_TICKET_TO_CLOSE_FOR_ORGANIZATION_TASK, defaultLimit
+        null, MAX_COUNT_COMPLETED_TICKET_TO_CLOSE_FOR_ORGANIZATION_TASK, defaultLimit
```

Comment:
why you don't need to send context?

Reply:
We really need caching of flags when they are requested within the same request so that the values are the same.   But when we request feature flags in tasks, then in the case of passing the Keystone context, we will always receive outdated data, except for the very first request, since the Keystone context retains the value of the feature flags.